ethsigner/app/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this still needed now that the log4j2.xml has been added to the core module?

Reply:
nope - removed.